src/test/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/SurveyLaunchedServiceImplTest.java

```diff
@@ -52,49 +94,30 @@ public void testSurveyLaunchedAddressAgentIdValue() throws Exception {
     assertEquals(surveyLaunchedDTO.getAgentId(), eventPayload.getAgentId());
   }
 
+  @Test
+  public void testSurveyLaunchedAddressAgentIdValue() throws Exception {
+    callAndVerifySurveyLaunched(Channel.AD);
```

Comment:
Nitpick but I think that the verify-survey-launched would have been better in a separate method. Don't see what is gained by mashing them together. 
If separate it would have allowed the methods to verify the results in the order that things happen. ie, verify rate limiter call before checking the sent event.

Reply:
I think the code reflects the order that things were added rather than the ordering that might be done if it was all in m head before it was written. Will be sure to try to think more about this going foward!